### PR TITLE
Provider for laravel 5.1 (and more)

### PR DESCRIPTION
**Pull request by damianopetrungaro**
Just update the new provider laoder for laravel 5.1 (and more).

### CODE REVIEW

**Comment by felixkiss** [comment]
`Felixkiss\UniqueWithValidator\UniqueWithValidatorServiceProvider::class` is perfectly fine for any Laravel version. Can you change your commit to just replace the old version with the new one?

**Comment by damianopetrungaro** [comment]
Sure!

File: README.md [hunk 1]
```diff
@@ -30,11 +30,12 @@ composer require felixkiss/uniquewith-validator:2.*
 
 Add the following to your `providers` array in `config/app.php`:
 
+
```
Comment (felixkiss) [inline README.md]: Added line break not necessary

File: README.md [hunk 2]
```diff
@@ -30,11 +30,12 @@ composer require felixkiss/uniquewith-validator:2.*
...
 ```php
 'providers' => array(
     // ...
 
-    'Felixkiss\UniqueWithValidator\UniqueWithValidatorServiceProvider',
+    Felixkiss\UniqueWithValidator\UniqueWithValidatorServiceProvider::class,
```
Comment (felixkiss) [inline README.md]: Please remove added line break and restore indentation

**Comment by felixkiss** [comment]
Can you squash your commits to one when you are finished?

**Pull request closed without merging**